src/D2L.CodeStyle.Analyzers/Immutability/ExplicitImmutableAttributeTransitivityAnalyzer.cs

```diff
@@ -144,6 +159,7 @@ string otherThingKind
 				declSyntax.Identifier.GetLocation(),
 				properties: FixArgs,

```

Comment:
These FixArgs still strictly suggest adding `[Immutable]`. Could be an improvement down the line to have a fix suggestion for either attribute